package.json

```diff
@@ -8,11 +8,11 @@
   "license": "ISC",
   "scripts": {
     "blockchain": "ganache --wallet.mnemonic \"indicate swing place chair flight used hammer soon photo region volume shuffle\" -i 1337",
-    "blockchain:contracts": "node --experimental-modules ./tests/e2e/setup-contracts.mjs",
-    "blockchain:testrun": "concurrently -k -s first \"npm:blockchain\" \"npm:blockchain:contracts\"",
+    "blockchain:contracts:synpress": "node --experimental-modules ./tests/e2e/setup-contracts.mjs",
+    "blockchain:contracts:testcafe": "node --experimental-modules ./src/test/setup-contracts-testcafe.mjs",
     "build": "run-s check-types build:css build:app",
     "build:dev": "cross-env NODE_ENV=development npm run build",
-    "build:test": "cross-env NODE_ENV=test NET=goerli npm run build",
+    "build:test": "cross-env NET=local npm run build",
```

Comment:
why is NODE_ENV=test removed?

Reply:
ya right, should not be removed, must be my mistake, good catch. will put it back in.